### DISCUSSION TITLE
is there any way to hide menu themes??

### DISCUSSION THREAD

**Discussion by jotarezen** [aristocratos/btop#591] [Ideas]
I would like to know if there is any way to hide some themes that I don't use, I think it would be much better for those who like to use more than one.
if it doesn't have it and it's a feature that can be done, I would really appreciate it if someone more experienced could give me the way to do it.
I'm starting now in programming and using linux and would be happy to contribute.

ps: **sorry if it's already been covered.**

**Comment by deckstose** [comment]
You can remove the `.theme` files from `/usr/share/btop/themes` that you don't want and they won't be loaded on startup. But if you've installed btop via a package manager the files will re-appear after you update btop. It's also not recommended to manually edit/delete/move files handled by your package manager as it can cause problems on updates.

There is no option to hide any themes in the menu and I don't think it's worth implementing, as most people will choose a theme and forget about it
